### ISSUE TITLE
intent handling wording change 

### ISSUE THREAD

**Issue by ProfessorWendy** [actions-on-google/actions-builder-codelab-level-1#5]
Custom intent handling - looks like it was renamed to User intent handling (step 4)

**Comment by Fleker** [comment]
You mean the screenshot?

**Comment by wendychun** [comment]
Yes, screenshot and instructions to user - "Click the + (plus sign) in the Start scene next to Custom intent handling."
